src/main/java/com/zabawaba/reflector/Constructors.java

```diff
@@ -0,0 +1,100 @@
+package com.zabawaba.reflector;
+
+import java.lang.reflect.Constructor;
+import java.util.HashSet;
+
+/**
+ * Provides a set of methods to manipulate {@link Constructor}
+ *
+ * @author zabawaba
+ */
+public class Constructors {
+
+	private static Filter<Constructor<?>> ALL_CONSTRUCTORS = new Filter<Constructor<?>>() {
+		public boolean apply(Constructor<?> constructor) {
+			return true;
+		}
+	};
+
+	private Object obj;
+
+	public Constructors(Object obj) {
+		this.obj = obj;
+	}
+
+	/**
+	 * Get a constructor that has the given name
+	 *
+	 * @param constructorName
+	 *            The name of the constructor to look for
+	 * @return The constructor that has the given name
+	 *
+	 * @throws NoSuchMethodException
+	 *             If no constructor exists with the provided name
+	 */
+	public ReflectorConstructor get(String constructorName) throws NoSuchMethodException {
```

Comment:
Should add an overloaded function that takes in parameter count to differentiate from overloaded constructors.